framework/src/fvkernels/FVAdvection.C

```diff
@@ -18,12 +18,13 @@ FVAdvection::validParams()
   params.addClassDescription(
       "Residual contribution from advection operator for finite volume method.");
   params.addRequiredParam<RealVectorValue>("velocity", "Constant advection velocity");
-  MooseEnum advected_interp_method("average upwind", "upwind");
+  MooseEnum advected_interp_method("average upwind skewness-corrected", "upwind");
 
-  params.addParam<MooseEnum>("advected_interp_method",
-                             advected_interp_method,
-                             "The interpolation to use for the advected quantity. Options are "
-                             "'upwind' and 'average', with the default being 'upwind'.");
+  params.addParam<MooseEnum>(
+      "advected_interp_method",
+      advected_interp_method,
+      "The interpolation to use for the advected quantity. Options are "
+      "'upwind', 'average' and 'skewness-corrected', with the default being 'upwind'.");
```

Comment:
we dont need to list the options in the param description, moosedocs does it already

but it feels like skewness-correction should be a boolean to the kernels, even though for us it's a different interpolation method. 

when we ll have more options than average and upwind there, we ll want a boolean on skewness correction

Reply:
I had to add the skewness-corrected approach because the interpolation weights are different. We need to use the skewness-corrected weights to interpolate everything else to stay consistent in the scheme. Once I streamline the weights and interpolation approaches in the code, I'll make skewness correction a bool!

Reply:
@GiudGiud are you satisfied with the response here?